apps/emqx_ft/src/emqx_ft_api.erl

```diff
@@ -101,14 +105,44 @@ schema("/file_transfer/files/:clientid/:fileid") ->
                 )
             }
         }
+    };
+schema("/file_transfer") ->
+    #{
+        'operationId' => '/file_transfer',
+        get => #{
+            tags => [<<"file_transfer">>],
+            summary => <<"Get current File Transfer configuration">>,
+            description => ?DESC("file_transfer_get_config"),
+            responses => #{
+                200 => ?SCHEMA_CONFIG,
+                503 => emqx_dashboard_swagger:error_codes(
```

Comment:
From the [handler](https://github.com/emqx/emqx/pull/10986/files#diff-ff518fef9250fcff7aa319de265464f7cef644d41ced2f5aa2a538e6f99e9c5eR185), it seems like this response code is not expected.

Reply:
503 is returned by `check_ft_enabled` filter

Reply:
But this schema doesn't seem to use that filter :thinking:

Reply:
Good catch! 🎣